subsys/net/lib/nrf_cloud/src/nrf_cloud_fota.c

```diff
@@ -175,33 +175,32 @@ static int fota_settings_set(const char *key, size_t len_rd,
 
 static int load_fota_settings(void)
 {
-	int ret = settings_subsys_init();
+	static bool settings_loaded = false;
+	int ret = 0;
 
-	if (ret) {
-		LOG_ERR("Settings init failed: %d", ret);
-		return ret;
-	}
+	if (!settings_loaded) {
```

Comment:
`settings_subsys_init` already contains a check like this, so you don't need it here

Not that I foresee it being a big problem, but you could get rid of it, and that would let this implementation remain flexible in case, for instance, Zephyr ever allows the settings subsystem to be **_unloaded_**

Reply:
this is more for `settings_load_subtree` which actually loads the data into `saved_job` via `fota_settings_set`

Reply:
Ahh, my bad. That makes sense.